cmd/xdpcap/testdata/test.c

```diff
@@ -0,0 +1,14 @@
+#include <linux/bpf.h>
```

Comment:
If we build real ELFs we have to worry about how to update them in the future:
- Which clang version should we use, and what magic flags.
- Where do the `<bpf/*>` headers come from.

In this case I think it's easier to just use asm and generate a program directly:

```
	entrypoint, err := ebpf.NewProgram(&ebpf.ProgramSpec{
		Type: ebpf.XDP,
		Instructions: asm.Instructions{
			// Context is already in R1.
			asm.LoadMapPtr(asm.R2, hookMap.FD()),
			asm.Mov.Imm(asm.R3, int32(xdpPass)),
			asm.FnTailCall.Call(),
			asm.Mov.Imm(asm.R0, int32(xdpPass)),
			asm.Return(),
		},
	})
	if err != nil {
		t.Fatal(err)
	}
```

Reply:
yeah that's a good idea. fixed